src/server/models/parking-lot.ts

```diff
@@ -0,0 +1,31 @@
+import { Document, Schema, model, Model } from 'mongoose';
+
```

Comment:
Here you are missing `Thruway` namespace. You have to bring it into scope before using it as now it is being exported.

```suggestion

import type { Thruway } from '../../@types/thruway';

```